db/migrate/20240604171601_drop_table_dish_ingredients.rb

```diff
@@ -0,0 +1,5 @@
+class DropTableDishIngredients < ActiveRecord::Migration[7.1]
```

Comment:
Wondering why there is a drop table here, I don't see that table having been created anywhere